core/src/main/java/org/infinispan/cache/impl/SimpleCacheImpl.java

```diff
@@ -923,6 +925,15 @@ public V remove(Object key) {
       return oldEntry != null ? oldEntry.getValue() : null;
    }
 
+   @Override
```

Comment:
do we have some tests on this ?

Reply:
Great question. It seems that we don't have a test querying a simple cache. According to the documentation the simple cache cannot be indexed, but maybe it could be queryable...

Reply:
Actually it seems that a simple cache cannot be queryable with the current API. So I think we should do the same with the new API. I'm adding a change to present a proper message to the user.

Reply:
done! thank you @karesti